src/client/src/app/site/[site]/view-availability/view-daily-appointments/view-daily-appointments-page.tsx

```diff
@@ -0,0 +1,98 @@
+import { Table, Pagination } from '@nhsuk-frontend-components';
+import { PaginationLink } from '../../../../lib/components/nhsuk-frontend/pagination';
```

Comment:
Could we stick this in `/components/nhsuk-front-end/index.tsx` then we can import it from the same place as _Table & Pagination_ on the line above as well.